apps/balancer-tools/src/app/apr/(utils)/chainsPublicClients.ts

```diff
@@ -0,0 +1,45 @@
+import { createPublicClient, http } from "viem";
+import {
+  arbitrum,
+  avalanche,
+  base,
+  gnosis,
+  mainnet,
+  polygon,
+  polygonZkEvm,
+} from "viem/chains";
+
+type ChainType =
+  | typeof mainnet
+  | typeof polygon
+  | typeof polygonZkEvm
+  | typeof arbitrum
+  | typeof avalanche
+  | typeof gnosis
+  | typeof base;
+
+export type ChainName =
+  | "ethereum"
+  | "polygon"
+  | "polygon-zkevm"
+  | "arbitrum"
+  | "avalanche"
+  | "gnosis"
+  | "base";
```

Comment:
couldn't we use the enums we already have? :/

Reply:
I tried but I got so complicated with the types that I decided to use this to make it work